lib/bibliothecary/analyser.rb

```diff
@@ -100,11 +100,18 @@ def analyse_contents(filename, contents)
             platform: platform_name,
             path: filename,
             dependencies: dependencies,
-            kind: determine_kind(filename)
+            kind: determine_kind(filename),
+            success: true
           }
         end
-      # rescue
-      #   nil
+      rescue
+        {
+          platform: platform_name,
+          path: filename,
+          dependencies: nil,
+          kind: determine_kind(filename),
```

Comment:
Can this throw an error? Actually looking below, it doesn't look like it but the case where nothing matches seems like it's weird as kind is a string and not an array type?

Reply:
So kind is a string in the success case? I'm not really certain what we want to happen there I'll dig into that a little more.